tests/test_alchemy.py

```diff
@@ -287,11 +278,20 @@ class SQLAlchemySessionFactoryTestCase(unittest.TestCase):
     def test_create_get_session_from_sqlalchemy_session_factory(self):
         SessionGetterFactory.create()
```

Comment:
Can also move the `SessionGetterFactory` declaration to this test. Up to you.

Reply:
And finally, done in [3dd6bb3](https://github.com/FactoryBoy/factory_boy/pull/927/commits/3dd6bb3d8d639ac29ab915576c218e4bd357f159) :)